sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/HardcodedIpAddressTest.cs

```diff
@@ -19,8 +19,10 @@
  */
 
 extern alias csharp;
+extern alias vbnet;
```

Comment:
why do we use this?

Reply:
We introduced this because we needed to test classes that have the same name and namespace in the VB and C# projects... If I remember correctly we decided to use the aliases for all `using`s to avoid confusion. I admit it is not always needed, for example in this particular test class we can very well work without the alias.